docs/server-side/v2/api-reference/external-streams/start-rtmp-for-a-given-room.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Start RTMP Streaming and / or recording for a given room
```

Comment:
Will reframe this to be only for RTMP. While this can be used for "just recording", it's a point of confusion since recording is much more popular and will have a separate API